### ISSUE TITLE
faild loading help_main from MAIN,cause = dlsym(RTLD_MAIN_ONLY,help_main):symbol not found

### ISSUE THREAD

**Issue by iT-Boyer** [blinksh/blink#2055]
### Checklist

- [ x ] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [ x ] Read the [docs](http://blink.sh/docs/)
- [ x ] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

blink:V18.0.0.927
ios:18.1 Beta 4
### Describe the bug

faild loading help_main from MAIN,cause = dlsym(RTLD_MAIN_ONLY,help_main):symbol not found

**Comment by unoctanium** [comment]
Same here. This is my workaround for this issue:

I get this warning in the debugger:

```
warning: (arm64) /Users/me/Library/Developer/Xcode/DerivedData/Blink-aydbomiivtincohlhljsdxjmwkyo/Build/Products/Debug-iphoneos/Blink.app/Blink empty dSYM file detected, dSYM was created with an executable with no debug info.
```

I tried to change Build Settings>Build Options>Debug Information Format, make sure both Debug and Release is 
- “DWARF” only for ALL build targets (Blink, Blink tests, …), or 
- “DWARF with dSYM File”. 
- The warning disappeared (I forgot in which case of those two), but after starting blink app and typing `help`, in both test cases I got this message in the terminal:  `Failed loading help_main from MAIN, cause = dlsym(RTLD_MAIN_ONLY, help_main): symbol not found`.

My workaround: In Xcode, Product | Scheme | Edit Scheme:
For **all** build targets, I selected “Run” in the Nav bar of the dialog and changed build configuration to “release”

And then I run the project again with Build target “Blink” and my device connected and selected  as the target in Xcode.

Now, the apps runs without warnings, and with all commands in the blink shell working.
